src/Elasticsearch.Net/Domain/RequestParameters/RequestParameters.Generated.cs

```diff
@@ -7001,6 +7001,40 @@ public class DeleteRoleRequestParameters : FluentRequestParameters<DeleteRoleReq
 		
 	}
 	
+	///<summary>Request parameters descriptor for XpackSecurityDeleteRoleMapping
+	///<pre>
+	///https://www.elastic.co/guide/en/x-pack/master/security-api-role-mapping.html#security-api-delete-role-mapping
+	///</pre>
+	///</summary>
+	public class DeleteRoleMappingRequestParameters : FluentRequestParameters<DeleteRoleMappingRequestParameters> 
```

Comment:
I think this should adhere to allowing a 404 status code response, similar to other Delete operations